### DISCUSSION TITLE
ufw firewall rules to run the self-hosted option

### DISCUSSION THREAD

**Discussion by improvethings** [coollabsio/coolify#3076] [Feature Requests]
Does anyone have a good ufw rule set for running your own self-hosted coolify server behind nginx with nothing else running?
I have nginx running (only on port 444, no 80)
 
$ sudo ufw status
Status: active

To                         Action      From
--                         ------      ----
22/tcp                     ALLOW       Anywhere
Nginx HTTPS                ALLOW       Anywhere
Anywhere on venet0         ALLOW       Anywhere
Anywhere on br-016745fea45 ALLOW       Anywhere
6001                       ALLOW       Anywhere
8000                       ALLOW       Anywhere

I still keep getting the error: 
_Coolify could not connect to its real-time service.
This will cause unusual problems on the UI if not fixed!_ 
but nothing showing up in syslog or the docker logs...

**Comment by v11t** [comment]
just fyi coolify (docker) doesn't respect ufw rules unless you mess with iptables or use something like https://github.com/chaifeng/ufw-docker
